CMakeLists.txt

```diff
@@ -42,4 +42,5 @@ set(FREERTOS_PORT_DIRECTORY ${BOARD_DEMO_FREERTOS_PORT_PATH} CACHE STRING "FreeR
 include_directories(${BOARD_DEMO_PORT_PATH})
 
 # Add middleware
+set(USE_COREHTTP ON)
```

Comment:
just out of curiosity, does this cause problems with the samples that don't use ADU?

Reply:
(sorry didn't mean to block hit the wrong button)

Reply:
Not really. This only causes azure-iot-middleware-freertos to build the http library (which uses coreHTTP). Samples that do not use the http interface will not leverage it, and will not be affected at all.